projects/wc/src/app/components/generic-ui/list-view/list-view.component.html

```diff
@@ -0,0 +1,36 @@
+<ui5-dynamic-page>
+  <ui5-dynamic-page-title slot="titleArea">
+    <ui5-title slot="heading">{{ heading }}</ui5-title>
+    <ui5-text slot="subheading">
+      This page displays the created {{ plural }} in your environment</ui5-text
+    >
+  </ui5-dynamic-page-title>
+  <ui5-table>
+    <ui5-table-header-row slot="headerRow">
+      @for (column of columns; track column.label) {
+        <ui5-table-header-cell>{{ column.label }}</ui5-table-header-cell>
+      }
+    </ui5-table-header-row>
+    <ui5-illustrated-message
+      slot="nodata"
+      name="NoData"
+    ></ui5-illustrated-message>
+    @for (item of items; track item.metadata.name) {
+      <ui5-table-row>
+        @for (column of columns; track column.label) {
+          @if (column.property === 'metadata.name') {
+            <ui5-table-cell>
+              <span fd-link (click)="navigateToAccount($event, item)">{{
```

Comment:
```suggestion
              <span fd-link (click)="navigateToResource($event, item)">{{
```

Maybe let's keep the name here already generic and not tie it to a specific account

Reply:
done